pubspec.yaml

```diff
@@ -32,10 +33,12 @@ dev_dependencies:
   json_serializable: 6.6.1
   retrofit_generator: 6.0.0+1
   spider: 4.2.1
-  surf_lint_rules: 2.1.0
+  surf_lint_rules: 3.0.0
 
 environment:
-  sdk: ">=2.17.0-0 <3.0.0"
+    sdk: '>=3.0.5 <4.0.0'
+    flutter: ">=3.0.0"
```

Comment:
Set concrete version of flutter `3.10.6`

Reply:
\+